starters/basic-starter/lib/get-prioritized-static-paths.ts

```diff
@@ -0,0 +1,43 @@
+import { drupal } from './drupal';
+
+// Generates paths of pages to pre-render with prioritization of menu links.
+export async function getPrioritizedStaticPathsFromContext(
+  context,
+  ENTITY_TYPES,
```

Comment:
I think we could rename this to just `types` since this is just a local variable